### ISSUE TITLE
[!] Invalid `Dangerfile` file: undefined method `header'

### ISSUE THREAD

**Issue by angelolloqui** [orta/danger-junit#27]
Starting today (with version 1.0.1) I get the following error when running Danger with a junit report on an iOS project.

```
bundler: failed to load command: danger (/Users/vagrant/.rbenv/versions/2.6.5/bin/danger)
Danger::DSLError: 
[!] Invalid `Dangerfile` file: undefined method `header' for #<Danger::Dangerfile:0x00007fba2c314c08>
 #  from Dangerfile:12
 #  -------------------------------------------
 #          markdown(message)
 >          junit.report        
 #      else 
 #  -------------------------------------------
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:70:in `method_missing'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/plugin_support/plugin.rb:23:in `method_missing'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-junit-1.0.1/lib/junit/plugin.rb:174:in `report'
  Dangerfile:12:in `processTestFiles'
  Dangerfile:32:in `block in eval_file'
  Dangerfile:31:in `each'
  Dangerfile:31:in `eval_file'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:307:in `eval'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:307:in `eval_file'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:200:in `block in parse'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:197:in `instance_eval'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:197:in `parse'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/dangerfile.rb:283:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/danger_core/executor.rb:29:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/lib/danger/commands/runner.rb:73:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
  /Users/vagrant/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/danger-8.0.2/bin/danger:5:in `<top (required)>'
  /Users/vagrant/.rbenv/versions/2.6.5/bin/danger:23:in `load'
  /Users/vagrant/.rbenv/versions/2.6.5/bin/danger:23:in `<top (required)>'
[!] The exception involves the following plugins:
 -  danger-junit
```

I am running Danger as part of a Bitrise step as follows:

```
def processTestFiles(path, name) 
    if !Dir.glob(path).empty? then
        junit.parse path
        message = "## #{name}\n" <<
            "Status | Number of tests |\n" <<
            "--- | --- | \n" <<
            " ✅ | #{junit.passes.count} tests passed |\n"
        if junit.failures.count > 0 then 
            message = message << " 🚫 | #{junit.failures.count} tests failed |\n"
        end
        markdown(message)
        junit.report        # <--- Fails here
    else 
        markdown("## #{name}\n⁉️: No tests run")    
    end
end

......

Dir["build/split-results/*"].each do |f| 
    processTestFiles(f, File.basename(f, ".*"))    
end

```

It was working OK in version 1.0.0

**Comment by aiKrice** [comment]
Can you post the whole junit part of your Dangerfile plz .
I will try to fix it asap. It seems I introduced regression with the new optional param
Sorry for inconvenience 🙏

**Comment by aiKrice** [comment]
@angelolloqui Fixed in #28. I apologized for this 🙏

**Comment by angelolloqui** [comment]
Thanks for the quick response!
